engine/src/cubos/engine/collisions/broad_phase/sweep_and_prune.cpp

```diff
@@ -3,6 +3,7 @@
 #include "sweep_and_prune.hpp"
 
 using cubos::core::ecs::Entity;
```

Comment:
```suggestion
```

Reply:
Can't do this because `sweep_and_prune.hpp` doesn't include `cubos.hpp` which includes `prelude.hpp`

Reply:
Why not include prelude.hpp then?

Reply:
Thought only `cubos.hpp` should include `prelude.hpp`, but ok I will include it!